.circleci/config.yml

```diff
@@ -2,38 +2,20 @@ version: 2.1
 
 orbs:
   docker: circleci/docker@2.1.3
-
-cache-key: &cache-key
-  key: dependency-cache-primary-{{ arch }}-yarn-packages-{{ checksum ".nvmrc" }}-{{ checksum "yarn.lock" }}
-
-commands:
-  yarn_install:
-    steps:
-      - restore_cache: *cache-key
```

Comment:
Did we kill the caching because it is coming by default now from pkg-manager?

Reply:
Yes it's built into the orb.